test.cc

```diff
@@ -275,3 +277,192 @@ TEST(Sherlock, SubscribeToStreamViaHTTP) {
   // TODO(dkorolev): Unregister the exposed endpoint and free its handler. It's hanging out there now...
   // TODO(dkorolev): Add tests that the endpoint is not unregistered until its last client is done. (?)
 }
+
+struct IntKey {
```

Comment:
I'm not sure we need this complexity in the test. Just `int key;` in `KeyValueEntry` might just do it.

(On the other hand, we need to start thinking of parsing `key` from URL parameters, for further REST-ful `GET`-s...)